test/externalTests/activateItem.js

```diff
@@ -0,0 +1,24 @@
+const assert = require('assert')
+
+module.exports = () => async (bot) => {
+  // Test that activateItem works without throwing errors
+  // This reproduces the issue where activateItem fails on 1.21+ due to packet format changes
+
+  const Item = require('prismarine-item')(bot.registry)
+
+  // Set up an item that can be activated
+  await bot.test.setInventorySlot(36, new Item(bot.registry.itemsByName.bread.id, 1, 0))
+  await bot.test.becomeSurvival()
+
+  // Test that activateItem works and doesn't throw errors
+  assert.ok(!bot.usingHeldItem, 'Should not be using item initially')
+
+  // This should work on all versions including 1.21+
+  await assert.doesNotReject(async () => {
```

Comment:
Pretty sure this does not check anything 

You must check that activate item has some action on the world that we can check with other mineflayer state or methods